core-sitemaps.php

```diff
@@ -18,3 +18,7 @@
  */
 
 // Your code starts here.
+
+require_once __DIR__ . '/inc/sitemaps-index.php';
```

Comment:
Let's rename this file to class-sitemaps-index.php, per https://engineering.hmn.md/standards/style/php/#file-naming.